app/assets/stylesheets/layout_components/sections.scss

```diff
@@ -201,6 +201,10 @@
     label{
       color: $white;
     }
+
+    input[type="radio"] + label{
+      color: $stormcloud;
+    }
```

Comment:
would this also change the text on the label itself? or just within the radio button?